src/Aspire.Hosting.Garnet/GarnetBuilderExtensions.cs

```diff
@@ -0,0 +1,147 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Globalization;
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Garnet;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding Garnet resources to the application model.
+/// </summary>
+public static class GarnetBuilderExtensions
+{
+    private const string GarnetContainerDataDirectory = "/var/garnet/data";
```

Comment:
Who reads/writes to this location? Did we find documentation somewhere that points to this location?

I see that Redis uses `/data`, and I've found that [in the Garnet code](https://github.com/microsoft/garnet/blob/918775dd88c10e7b6a8ec4f200f91a0de48dedcf/docker-compose.yml#L12) as well.

Should this be:

```suggestion
    private const string GarnetContainerDataDirectory = "/data";
```
?

Reply:
I'll check this evening

Reply:
Really. the documentation itself states exactly /data
But both options work.
I'll change it to /data this evening